tests/contrib/anthropic/conftest.py

```diff
@@ -42,6 +42,7 @@ def mock_tracer(ddtrace_global_config, anthropic):
         LLMObs.disable()
         LLMObs.enable(_tracer=mock_tracer, integrations_enabled=False)
     yield mock_tracer
+    LLMObs.disable()
```

Comment:
try/finally?

what happens if we don't call disable?

Reply:
added the try finally, I think we would end up with some additional span tags, but I'm not 100% sure how the LLMobs integrations work

Reply:
LLMObs not being disabled has a possibility to leak LLMObs behavior into test cases that do not use LLMObs (i.e. the remaining anthropic tests). This would result in llmobs functionality being run in the integration which we do not want